utilities_common/platform_sfputil_helper.py

```diff
@@ -93,3 +103,34 @@ def get_interface_alias(port, db):
                 sys.exit(1)
 
     return port
+
+
+def is_rj45_port(port_name):
+    global platform_sfputil
+    global platform_chassis
+    global platform_sfp_base
+    global platform_sfputil_loaded
+
+    if not platform_chassis:
+        import sonic_platform
+        platform_chassis = sonic_platform.platform.Platform().get_chassis()
+    if not platform_sfp_base:
+        import sonic_platform_base
+        platform_sfp_base = sonic_platform_base.sfp_base.SfpBase
+
+    if platform_chassis and platform_sfp_base:
+        if not platform_sfputil:
+            load_platform_sfputil()
+
+        if not platform_porttab_mapping_read:
```

Comment:
can we move this function to sfputil/main.py ...that would make it much simpler

Reply:
it will be called from sfpshow, intfutil as well. in those files, we need to handle `RJ45` port differently even if it is absent from `STATE_DB`.